src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/HiddenBackpack.java

```diff
@@ -0,0 +1,14 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.tools;
+
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import org.bukkit.inventory.ItemStack;
+
+public class HiddenBackpack extends SlimefunBackpack {
```

Comment:
Should maybe rather be called "RestoredBackpack". Also please add adequate javadocs here.